src/components/SpecialResolution/CreateSpecialResolution.vue

```diff
@@ -370,15 +393,15 @@ export default class CreateSpecialResolution extends Mixins(DateMixin) {
 
   /** Set validate on file and pay click. */
   @Watch('getComponentValidate')
-  protected updateResolutionValidationDetail (): void {
+  private updateResolutionValidationDetail (): void {
```

Comment:
Typically, watchers are private since they're not called externally or from template.